phpunit.xml.dist

```diff
@@ -9,7 +9,7 @@
          processIsolation="false"
          stopOnFailure="false"
          syntaxCheck="false"
-         bootstrap="tests/bootstrap.php"
+         bootstrap="tests/tests/bootstrap.php"
```

Comment:
I agree, but I checked admin, media, classification - all have the same structure `bootstrap="Tests/tests/bootstrap.php"`   ¯\\_(ツ)_/¯
Thats is also true for above note, all these bundles have autoload-dev with tests in composer.json
I suppose that's because namespaces in tests are not PSR-4 compliant